book/platform/examples/correct/hello-world/lib/hello.mli

```diff
@@ -0,0 +1,27 @@
+(* This file is the interface (aka exposed API, or signature) for the [Hello]
+   module. It only serves as documenting the module and selecting the functions
+   to expose. The actual implementation of the functions are located in the
+   module implementation [hello.ml]. *)
+
+(** This is a docstring, as it starts with "(**", as opposed to normal comments
+    that start with "(*".
+
+    The top-most docstring of the module should contain a description of the
+    module, what it does, how to use it, etc.
+
+    The function-specific documentation located below the function signatures. *)
+
+val greet : string -> string
+(** This is the docstring for the [greet] function.
+
+    You can read more about the ocamldoc syntax used to write docstrings here:
+
+    https://ocamlverse.github.io/content/documentation_guidelines.html
```

Comment:
https://ocaml.github.io/odoc/odoc_for_authors.html :-)

Though having said that the ocamlverse site _is_ pretty nice :-)